CHANGELOG.md

```diff
@@ -51,6 +51,10 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ### Removed
 - It is not longer possible to choose to convert HTML sub- and superscripts to equations
 - [#1610](https://github.com/JabRef/jabref/issues/1610) Removed the possibility to auto show or hide the groups interface
+- Removed option to open right-click menu with ctrl + left-click as it was not working
```

Comment:
I could have split it is two, as have separate ones would be a conflict heaven, especially considering that I had to manually renumber the layout of the preference tab where four entries were removed.

What I could have done though was to not group the preference keys.

The best way to review is to check the non-JabRefPreferences code, especially the preference tabs. You will notice that the key removed there does not exist in any of the other files (except JabRefPreferences). This covers most changes (GENERAL_FIELDS) is the other, as I recall it.